packages/libs/react-ui/src/components/Form/Form.css.ts

```diff
@@ -48,10 +48,20 @@ export const baseContainerClass = recipe({
     {
       transition: 'outline-color 0.2s ease-in-out',
       outlineColor: 'transparent',
+      minWidth: '150px',
+      '@media': {
+        [breakpoints.sm]: {
+          maxWidth: '100%',
+        },
+        [breakpoints.md]: {
+          maxWidth: '40rem',
+        },
+      },
```

Comment:
I would recommend using the `responsiveStyles` utility that we export for this

Reply:
Changed in the next commit.